ckan/templates/snippets/csrf_input.html

```diff
@@ -1,2 +1,8 @@
 {# Adds a csrf_token to the forms as a hidden input. #}
+{%- if refresh_time -%}
+  <div hx-get="{{ h.url_for("util.csrf_input") }}" hx-trigger="load delay:{{ refresh_time }}s" hx-swap="outerHTML">
+{%- endif -%}
 <input type="hidden" name="{{ g.csrf_field_name }}" value="{{ csrf_token() }}"/>
```

Comment:
We could look at placing a 'not-after' meta on this field instead of a delay timer so if cached it will still get a fresh token for usage.

> Should Timestamps be Included in CSRF Tokens for Expiration?
>
>It's a common misconception to include timestamps as a value to specify the CSRF token expiration time. A CSRF >Token is not an access token. They are used to verify the authenticity of requests throughout a session, using >session information. A new session should generate a new token ([1](https://stackoverflow.com/a/30539335)).

https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html